tests/Extensions/TimedScopes.UnitTests/ActivityObserversIntializerTests.cs

```diff
@@ -50,8 +54,15 @@ public async Task ActivityObserversInvokedProperly()
 			}
 		}
 
+		private void AssertEnabledFor(DiagnosticListener listener, string eventName) =>
+			Assert.IsTrue(listener.IsEnabled(eventName), "Should be enabled for '{0}'", eventName);
+
 		private string MakeStartName(string name) => name + ".Start";
```

Comment:
You could use the consts you defined in the beginning here and in the next line: ActivityStartEnding and ActivityStopEnding. You can also group all const string  in one place.

Reply:
I don't like to make constants public, we could probably make them internal but it still having internal constant sounds wrong, but I could probably change it to readonly string

Reply:
Switched to internal static readonly